src/main/java/com.marketdata.api/MarketDataController.java

```diff
@@ -1,7 +1,5 @@
 package com.marketdata.api;
 
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
```

Comment:
Removed these unused imports. [https://stackoverflow.com/questions/979057/any-reason-to-clean-up-unused-imports-in-java-other-than-reducing-clutter](This link explains the reasoning for removing unused imports.) Two imports won't affect performance, but it can help to alleviate issues down the road. Only import things as needed!

Reply:
https://www.baeldung.com/java-clean-code

Clean code is important!

Reply:
Good idea.  The IDE can do this for you automatically with Control Shift O.  It's good to do that periodically... though it does not necessarily require a new branch.